### ISSUE TITLE
[2.3] Error on checkout/payment in PHP 5.2.17 in class-wc-emails.php

### ISSUE THREAD

**Issue by jconroy** [woothemes/woocommerce#7270]
Using a fresh install of WP 4.1 with WC 2.3 (master branch) and the WC dummy data (no plugins) when I go to pay/checkout I get the following error

```
Fatal error: func_get_args(): Can't be used as a function parameter in /woo/wp-content/plugins/woocommerce/includes/class-wc-emails.php on line 87
```

Here's the trace if useful http://cl.ly/image/3C3A3T3u1l2l

---

PHP 5.5.3 doesn't appear to have the issue. Thought I'd report for confirmation / checking

**Comment by bryceadams** [comment]
Thanks @jconroy - can you please try replace line 87 of that file with:

```
$args = func_get_args();
do_action_ref_array( current_filter() . '_notification', $args );
```

And confirm it's fixed?

**Comment by jconroy** [comment]
@bryceadams yep appears to work

**Comment by bryceadams** [comment]
@jconroy excellent, thanks for checking. This will be fixed for 2.3.
